README.md

```diff
@@ -42,7 +40,7 @@ First clone the Repository
 Use python 3 instead of Python 2
 
 ```
-  $ ./setup.sh
+  $ sudo ./setup.sh
```

Comment:
Any reason for `sudo`?

Reply:
Hi @rajikaimal bro, 
We have a couple of commands during dependency installation which required to be run as the superuser. Instead of doing it multiple times for each command, simply running the `./setup.sh`  file as superuser would solve the hassle.

Reply:
Oh okay 👍